tools/dexnode/src/hosts/msedge.ts

```diff
@@ -0,0 +1,104 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import path from "path";
+import { type Host } from "../host.js";
+import { HostFlags } from "../hostFlags.js";
+
+export const HOST_MSEDGE_STABLE: Host<"msedge"> = {
+    flagName: "msedge",
+    flags:
+        HostFlags.ChromiumArguments |
+        HostFlags.UseLatestV8Version,
+    paths: {
+        win32: [
+            "C:/Program Files/Microsoft/Edge/Application/msedge.exe",
+            "C:/Program Files (x86)/Microsoft/Edge/Application/msedge.exe",
+        ],
+        darwin: [
+            "/Applications/Microsoft Edge.app/Contents/MacOS/Microsoft Edge",
+        ],
+        linux: [
+            "/opt/microsoft/msedge/msedge",
+            "/usr/bin/microsoft-edge-stable",
+            "microsoft-edge-stable"
+        ],
+    },
+    registry: [
+        { key: "SOFTWARE\\Clients\\StartMenuInternet\\Microsoft Edge\\shell\\open\\command" },
+        { key: "SOFTWARE\\Wow6432Node\\Clients\\StartMenuInternet\\Microsoft Edge\\shell\\open\\command" },
+    ],
+};
+
+export const HOST_MSEDGE_DEV: Host<"msedge_dev"> = {
+    flagName: "msedge_dev",
+    flags:
+        HostFlags.ChromiumArguments |
+        HostFlags.UseLatestV8Version,
+    paths: {
+        win32: [
+            "C:/Program Files/Microsoft/Edge Dev/Application/msedge.exe",
+            "C:/Program Files (x86)/Microsoft/Edge Dev/Application/msedge.exe",
+        ],
+        darwin: [
+            "/Applications/Microsoft Edge Dev.app/Contents/MacOS/Microsoft Edge Dev",
+        ],
+        linux: [
+            "/opt/microsoft/msedge-dev/msedge-dev",

```

Comment:
I _think_ that for each of these, the binary won't have a suffix (just its parent dir), but, it probably doesn't hurt.